policy_packs/aws/ssm/enforce_install_packages_on_ec2_instances_using_ssm_package_installer/README.md

```diff
@@ -0,0 +1,205 @@
+---
+categories: ["security"]
+primary_category: "security"
+---
+
+# Enforce Install Packages on AWS EC2 Instances Using AWS SSM Package Installer
+
+Enforcing the installation of packages on EC2 instances using the AWS SSM Package Installer is crucial for maintaining a consistent, secure, and automated software deployment process. This measure ensures that all package installations are managed and logged through AWS Systems Manager, enhancing security, compliance, and operational efficiency by providing centralized control and monitoring of software installations.
+
+This [policy pack](https://turbot.com/guardrails/docs/concepts/resources/smart-folders) is designed to allow package installation on instances with or without direct access to original package source and can help you configure the following settings for SSM documents:
+
+- Set the region ARN where the SSM resources will be deployed
+- Set the IAM role ARN to be used by SSM to run documents
+- Create three SSM documents in the specified region
+  - Turbot-LinuxPackageInstaller-PublicNetwork
+  - Turbot-LinuxPackageInstaller-PrivateNetwork
+  - Turbot-LinuxS3PackageInstaller
+
+## Documentation
+
+- **[Review Policy settings →](https://hub-guardrails-turbot-com-git-development-turbot.vercel.app/policy-packs/enforce_install_packages_on_ec2_instances_using_ssm_package_installer/settings)**
+- Using the SSM documents
```

Comment:
I think I'm still not sure what I'd do with this info or why it's under Documentation. Documentation is normally something I'd read as reference material, but here there is none really, just our descriptions. Should we break these out into another H2 (or H3 under Getting Started somewhere) so it's more clear what these SSM docs are, where do I get them from, etc.?

Reply:
Makes sense. Add `### Using the SSM Documents` section under Getting Started.